build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -32,6 +32,7 @@ CONFIG_PASS1_TARGET="all"
 CONFIG_PASS1_OBJECT=""
 CONFIG_BOARD_BUILD_DATE="200204"
 # CONFIG_BINARY_SIGNING is not set
+CONFIG_DOCKER_VERSION="2.0.0"
```

Comment:
Please add the config using Kconfig. Otherwise it will disapper when someone uses menuconfig.

Reply:
Done